extract_unique_sensors_01.py

```diff
@@ -1,7 +1,10 @@
+import sys
 import argparse
+import logging
+
 from urllib.request import urlopen
 from pathlib import Path
-
+from logging import critical, error, info, warning, debug
```

Comment:
Mmhhh I think I should introduce you to linters, they'll help you write clean code.

Please check out this blogpost and implement it

Reply:
I ran flake8 to my script and cleaned all the errors and warnings.